src/views/Card/Card.js

```diff
@@ -0,0 +1,29 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+
+function Card(props) {
+  const { children, className, ...rest } = props
+  const classes = cx(
+    className,
```

Comment:
Add `ui` class as the first class.  Then, put the user's className as the last class.